.travis.yml

```diff
@@ -12,7 +12,6 @@ php:
 
 matrix:
   allow_failures:
-    - php: 7.1
```

Comment:
I've found what it does: https://secure.php.net/manual/en/ini.core.php#ini.zend.assertions

I've also found the only remaining usage of that: https://github.com/getsentry/sentry-php/blob/d34ab2569aa4841230f6125585c3fe426652137f/test/Raven/Tests/ClientTest.php#L13

I'll address this in a commit, so we can get rid of those additional build steps